nix/tullia.nix

```diff
@@ -0,0 +1,72 @@
+self: system:
+
+let
+  ciInputName = "GitHub event";
+in rec {
+  tasks = let
+    inherit (self.inputs.tullia) flakeOutputTasks taskSequence;
+
+    common = {
+      config,
+      ...
+    }: {
+      preset = {
+        # needed on top-level task to set runtime options
+        nix.enable = true;
+
+        github-ci = {
+          enable = config.actionRun.facts != {};
```

Comment:
what does this do? why is it set to what it is?

Reply:
Tullia tasks can run locally or on Cicero. When no facts are present we know that we are running locally and vice versa. When running locally, the current directory is already bind-mounted into the container, so we don't need to fetch the source from GitHub and we don't want to report a GitHub status.